### ISSUE TITLE
[Internationalization] AtOnboarding package

### ISSUE THREAD

**Issue by ande4485** [atsign-foundation/at_widgets#624]
### Which at_widget package you want to add enhancement?

at_onboarding_flutter

### On what platforms do you want to add enhancement?

Android, iOS, Windows, MacOS, Linux

### Is your feature request related to a problem?

I made an app in English and French but AtOnboarding widget is in English.
So I'm investigating to implement internationalization in this package and I created this issue to have your opinion.

### Describe alternatives that you have considered

I have 2 alternatives:
- I10n like https://docs.flutter.dev/development/accessibility-and-localization/internationalization. But in MaterialApp ( who use at_onboarding_flutter package) , you have to add 
 ```
const MaterialApp(
  title: 'Localizations Sample App',
  localizationsDelegates: [
    AtOnBoardingLocalizations.delegate,
    GlobalMaterialLocalizations.delegate,
    GlobalWidgetsLocalizations.delegate,
    GlobalCupertinoLocalizations.delegate,
  ],
  supportedLocales: [
    Locale('en'), // English
    Locale('fr'), // French
  ],
  home: MyHomePage(),
);
```
AtOnBoardingLocalizations.delegate it was what I tested( you can see it on screenshot, translated in french). It's quite simple and maintenable.

-  Map solution :
One array with all keys ( like 'yesString')
A map for each language ( like {'yesString':'yes', ...} for map_english, {'yesString':'oui'} for map_french)
And I can check the Locale value and if not set, english will be the dafault value.

What do you think ?
<img width="752" alt="Capture d’écran 2023-03-16 à 16 03 02" src="https://user-images.githubusercontent.com/38383248/225677784-1a5d12c6-cb76-4aeb-9b5e-3b46f3d475b0.png">

**Comment by sonle-geekyants** [comment]
I think that I will do this task with case 1. @ande4485

**Comment by sonle-geekyants** [comment]
@sachins-geekyants , Could you check my PR: https://github.com/atsign-foundation/at_widgets/pull/633 ?
@ande4485  Could you check out to this branch https://github.com/atsign-foundation/at_widgets/pull/633 and check it?

**Comment by ande4485** [comment]
@sonle-geekyants, I check quickly and there are some translation errors in french. I'll try to fix it this week.

**Comment by ande4485** [comment]
You'll find my modification : #644

**Comment by sonle-geekyants** [comment]
I've approved it. @ande4485

**Comment by sachins-geekyants** [comment]
Released with `at_onboarding_flutter: ^6.1.0`
